### DISCUSSION TITLE
Welded Plates (Osmiridium)

### DISCUSSION THREAD

**Discussion by JeremyALS** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1620] [General]
I need the energy of all living things in this universe just to weld a single Osmiridium plate using the arc welder 😭 DFC reactor gonna have to wait a few centuries or so

**Comment by Creeper-banner** [comment]
> I need the energy of all living things in this universe just to weld a single Osmiridium plate using the arc welder 😭 DFC reactor gonna have to wait a few centuries or so

An ICF can generate the energy you need. :-)

**Reply by hz0909adc** [reply]
Need a lot of boilers（
